recipe/meta.yaml

```diff
@@ -26,13 +26,13 @@ requirements:
     - tomli 2.0.1
   run:
     - python
-    - click >=7.0
-    - cloudpickle >=1.1.1
-    - fsspec >=0.6.0
+    - click >=8.0
+    - cloudpickle >=1.5.0
+    - fsspec >=0.9.0
```

Comment:
Should this be `>=2021.09`?

Reply:
I had the same question! Thanks for asking!

Reply:
So, I wasn't really sure since the previous version `2023.4.1` had `0.6.0` as the version, but I just checked the [upstream](https://github.com/dask/dask/blob/2023.4.1/pyproject.toml#L35) from the previous version and it has `>=2021.09.0` as well... 

But now I see that version `2023.4.0` has that `0.6.0` version for `fsspec` in the [upstream](https://github.com/dask/dask/blob/2023.4.0/pyproject.toml#L35) so I think I will change it.

Reply:
Agreed this should be changed.